csaf_2.1/prose/edit/src/distributing.md

```diff
@@ -288,6 +288,8 @@ having the `rel` value of `signature`.
 
 The use and therefore the existence of ROLIE service document is optional.
 If it is used, each ROLIE service document MUST be a JSON file that conforms with [cite](#RFC8322) and lists the ROLIE feed documents.
+It MAY also list the corresponding ROLIE category documents.
```

Comment:
I think that we often use MAY slightly different from what it is expected to signal.
My understanding of what we want to say here is that the listing of the corresponding ROLIE category documents is optional. Could we state this without a MAY?

Reply:
Addressed in [9de0c75](https://github.com/oasis-tcs/csaf/pull/745/commits/9de0c7581a939d119adf9d9305520be921302bfd)